src/docs/development/ui/advanced/splash-screen.md

```diff
@@ -115,8 +115,59 @@ to the normal theme at the appropriate time.
 The Android app now displays the desired launch screen
 while the app initializes.
 
+### Android S
+
+See [Android Splash Screens][] first on how to configure your splash screen on
+Android S.
+
+Make sure neither `io.flutter.embedding.android.SplashScreenDrawable` is set in
+your manifest, nor is `provideSplashScreen` implemented. Doing so will cause
+the Android splash screen to fade smoothly into the Flutter when the app is
+launched.
+
```

Comment:
Could indicate that `io.flutter.embedding.android.SplashScreenDrawable` is deprecated?  

I could add some logging to the embedding to notify that `io.flutter.embedding.android.SplashScreenDrawable` is deprecated.

Reply:
Done. Thanks for taking that, and we probably want to tag `provideSplashScreen` with `@Deprecated` too